worlds/alttp/docs/fr_A Link to the Past.md

```diff
@@ -0,0 +1,32 @@
+# A Link to the Past
+
+## Où se trouve la page des paramètres ?
+
+La [page des paramètres du joueur pour ce jeu](../player-settings) contient tous les paramètres dont vous avez besoin 
+pour configurer et exporter le fichier.
+
+## Quel est l'effet de la randomisation sur ce jeu ?
+
+Les objets que le joueur devrait normalement obtenir au cours du jeu ont été déplacés. Il y a tout de même une logique
+pour que le jeu puisse être terminé, mais dû au mélange des objets, le joueur peut avoir besoin d'accéder à certaines
+zones plus tôt que dans le jeu original.
+
+## Quels sont les objets et endroits mélangés ?
+
+Tous les objets principaux, les collectibles et munitions peuvent être mélangés, et toutes les zones du jeu qui
```

Comment:
I don't think "Zone" is a good translation for "Location" in this case, as an Archipelago location is not necessarily a place, it can be an action, etc.

I would propose "endroit" as a, hopefully, better translation, that encompasses both the places and the things like chests

Reply:
I wanted to keep it consistent with the above usage of "zones", but "endroit" does sound better in this case